### DISCUSSION TITLE
Waiting for check to complete...Firefox Error

### DISCUSSION THREAD

**Discussion by DrUmranAli** [zero01101/openOutpaint#121] [General]
Running with latest build of SD and the openOutpaint webui extention for auto1111, on firefox(nightly build 110.0a1 (2022-12-24) (64-bit). The --api flag in my webui file 

`set COMMANDLINE_ARGS= --xformers --api`

The extention loads up, but if you try and 'dream' it'll say offline in red, hovering over the host connection status it says 

Waiting for check to complete.

Everything works fine in Chrome/Edge....

**Comment by seijihariki** [comment]
Can you send us a screenshot of the network tab on devtools (F12)?
